packages/spotlight/Spottable/Spottable.js

```diff
@@ -334,12 +335,16 @@ const Spottable = hoc(defaultConfig, (config, Wrapped) => {
 
 		handleEnter = (ev) => {
 			forward('onMouseEnter', ev, this.props);
-			this.isHovered = true;
+			if (hasPointerMoved(ev.clientX, ev.clientY)) {
```

Comment:
Can we guarantee the ordering of events such that the move will always happen -after- the enter/leave?

Reply:
I can't find/link to a definitive spec, however in my experience across platforms (Chrome, FF, Edge, TV) the `mousemove` event always fires after the other related mouse events: `mouseover`, `mouseout`, `mouseenter`, and `mouseleave`

Reply:
OK!